pages/requests/[id].js

```diff
@@ -25,14 +25,20 @@ import {
 const Request = () => {
   const router = useRouter()
   const { data: session } = useSession()
+  // queries the router to get the quote group (request) id
   const { id } = router.query
   const { request, isLoadingRequest, isRequestError } = useOneRequest(id, session?.accessToken)
   const { allSOWs, isLoadingSOWs, isSOWError } = useAllSOWs(id, request?.identifier, session?.accessToken)
-  const { messages, isLoadingMessages, isMessagesError, mutate, data } = useAllMessages(id, session?.accessToken)
+  const { messages, 
+    files, 
+    isLoadingMessagesAndFiles, 
+    isMessagesAndFilesError, 
+    mutate, 
+    data } = useMessagesAndFiles(id, session?.accessToken)
```

Comment:
```suggestion
  const {
    messages, 
    files, 
    isLoadingMessagesAndFiles, 
    isMessagesAndFilesError, 
    mutate, 
    data,
  } = useMessagesAndFiles(id, session?.accessToken)
```

Reply:
we're now returning a lot of things from this function. may refactor this in the future.